src/lib/download/types/DataDownloadFilter.ts

```diff
@@ -0,0 +1,6 @@
+export interface DataDownloadFilter {
```

Comment:
Should this extend a `TimeSeriesFilter` (or something), since we are using it on the FEWS PI time series end point?